lib/index.ts

```diff
@@ -34,6 +34,7 @@ function getAllSwears(): Array<swearVariants> {
         [`charrue`],
         [`charogne`],
         [`gériboire`],
+        [`colon`, `gros colon`],
```

Comment:
Love `colon`, but I usually try to avoid adjective modifiers. Single-word versions fit better with other swears.

Reply:
✅ 12c0968